modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -31,9 +33,9 @@ spec:
 ----
 <1> Configures the Operator to create the Prometheus `ServiceMonitor` CR to scrape the Collector's internal metrics endpoint and Prometheus exporter metric endpoints. The metrics will be stored in the OpenShift monitoring stack.
 
-Alternatively, a manually created Prometheus `PodMonitor` can provide fine control, for example removing duplicated labels added during Prometheus scraping.
+Alternatively, a manually created Prometheus `PodMonitor` CR can provide fine control, for example removing duplicated labels added during Prometheus scraping.
```

Comment:
Since there is an alternative, a Procedure module with two choices might make this choice more clear: either create an Open Telemetry CR, or create a PodMonitor CR.

Regardless, even if this remains a Reference, I think it would be worth mentioning in the intro that there are two different ways to accomplish the same goal. That way, the reader doesn't get half way through before realizing that there might be a better way to do the thing they're trying to do.

Reply:
Thank you for the suggestion. I've added the following sentence to this section:

> One of two following custom resources (CR) configures the sending of metrics to the monitoring stack: the OpenTelemetry Collector CR or the Prometheus `PodMonitor` CR.